src/TritonRoute/src/dr/FlexGridGraph_maze.cpp

```diff
@@ -802,9 +802,34 @@ frMIdx FlexGridGraph::getUpperBoundIndex(const frVector<frCoord>& tracks,
       ||  // comment out for non-buffer enablement
       currGrid.getLastDir() == dir) {
     return false;
-  } else {
-    return true;
+  } 
+  if (ndr_) {
+    frCoord hw = (frCoord) design_->getTech()
```

Comment:
Totally optional, but I would consider renaming this halfWidth. I think it helps with readability especially those with those like myself who have less experience with the lingo of EDA software.

Reply:
What is the purpose of this line?  It looks like the value with either be unused or overwritten on 813.  I agree on the rename.

Reply:
The purpose is to get the half width to a variable, which will be compared to other width in the if below. Then the same variable will get another half width.